File: app/src/main/java/com/bitcoin/merchant/app/screens/legal/PrivacyPolicyFragment.java
package com.bitcoin.merchant.app.screens.legal;

import android.webkit.WebView;

import com.bitcoin.merchant.app.R;

public class PrivacyPolicyFragment extends LegalTextFragment {
    protected void setupWebView(WebView webView) {
        setToolbarTitle(R.string.menu_privacy_policy);
        webView.loadUrl("https://www.bitcoin.com/privacy-policy/");
    }
}

File: app/src/main/java/com/bitcoin/merchant/app/screens/legal/PrivacyPolicyFragment.kt
package com.bitcoin.merchant.app.screens.legal

import android.webkit.WebView
import com.bitcoin.merchant.app.R

class PrivacyPolicyFragment : LegalTextFragment() {
    override fun setupWebView(webView: WebView) {
        setToolbarTitle(R.string.menu_privacy_policy)
        webView.loadUrl(resources.getString(R.string.url_privacy_policy))
    }
}